types/fetch-meta-tags/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for fetch-meta-tags 1.0
+// Project: https://github.com/luisivan/fetch-meta-tags#readme
+// Definitions by: Alessandro Rabitti <https://github.com/silversonicaxel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface fetchedMeta {
+    url: string;
+    title?: string;
+    description?: string;
+    icon?: string;
+    image?: string;
+}
+
+/* tslint:disable-next-line:no-declare-current-package no-single-declare-module */
```

Comment:
```suggestion
```
Fix the error after removing the lint disable rule.

Reply:
@sheetalkamat fixed as you suggested